research/src/pages/select/select.proposal.mdx

```diff
@@ -183,6 +195,26 @@ and is therefore considered to be invalid.
 
 ### Interaction &amp; transition of states
 
+[Select Usage State Diagram](https://xstate.js.org/viz/?gist=7965135d58118c828140f6e830ca93a1)
+
+![Usage of select within other UI](../../images/statediagram-usage-select.png)
+
+[Select State Diagram](https://xstate.js.org/viz/?gist=24e03b6b21dcb0d1eb7471d6ff9043f3)
```

Comment:
@dandclark can comment further, but if we are mapping to `combobox` then we should likely follow the combobox interaction model rather than a single instance. I'm not following how this example represents the current state of select in Open UI.

Reply:
I can add flows for all of the keyboard interactions so it feels complete.